charts/common/values.yaml

```
@@ -82,9 +84,11 @@ service:
   #   # Specify the default port information
   #   port:
   #     port: ""
```

```suggestion
  #     port: 
```
Small leftover from when we had the fluxv2 issues